doc/api/http.md

```diff
@@ -1473,6 +1510,20 @@ It must be set to a non-zero value (e.g. 120 seconds) to protect against
 potential Denial-of-Service attacks in case the server is deployed without a
 reverse proxy in front.
 
+### `server.idleConnections()`
```

Comment:
This method makes sense to me. I think it is named correctly and returns the expected result.